src/modules/fstab/main.py

```diff
@@ -152,20 +152,22 @@ def generate_crypttab_line_info(self, partition):
         if not mapper_name or not luks_uuid:
             return None
 
-        if mount_point == "/":
+        if mount_point != "/":
```

Comment:
+1 to removing the `if` clause, I'm pretty sure we also want the `/` line in `crypttab` for `dracut`. (Please update your patch to do that, the current `!=` clause does not make sense.)